### DISCUSSION TITLE
Smite 1.1.0

### DISCUSSION THREAD

**Discussion by leon4rdodev** [thatskymod/Sky-CotL-Scripts#77] [Ideas]
Currently it's working well except for a few functionalities, can you add it to the libs folder?

**Comment by thatskymod** [comment]
I've heard a fair amount of positive feedback regarding **SMite v1.1.0**. While it looks promising on paper, its current state makes me hesitant to add it. The concern is that it suffers from crashes and unpredictable behavior. Adding it now could lead to a sudden influx of support requests, which I'm not prepared to handle given the incomplete nature of the project. Perhaps there's a way to label this version as unstable, but ideally, this repository is aimed at features that are thoroughly tested and fully functional.

**Comment by thatskymod** [comment]
**LibSmite 1.1.0** has been superseded by **LibSmite 1.2.0**, now available at:

https://github.com/thatskymod/Sky-CotL-Scripts/blob/a2cebcbcc3dd76a2ce1f3f6c4fb6dc4fa2ca5761/Canvas/Libs/libsmite-1.2.0.so

The previous version will no longer receive official support. Consider this thread resolved.
